x-pack/plugins/monitoring/public/less/components/logstash/config_viewer.less

```diff
@@ -0,0 +1,133 @@
+.cv-statementList
+{
+  margin: 16px;
+}
+
+.cv-statementList__icon
+{
+  margin-top: 5px;
+}
+
+.cv-spacer {
+  height: 94%;
+  width: 10px;
+  display: inline-block;
+  margin-left: 11px;
+  border-left: 1px #b2b2b2 dashed;
+}
+
+@media (max-width: 768px) {
+  .cv-spacer {
+    border: 0px;
+  }
+}
+
+.cv-spacer:last-child {
+  width: 0px;
+}
+
+.cv-inputMetric__eventsEmitted {
+  width: 150px;
+}
+
+.cv-statement__content {
+  display: inline-block;
+  min-height: 28px;
+  padding-left: 9px;
+  width: 100%;
+}
+
+.cv-statement {
+  display: inline-block;
+  margin-top: 5px;
+  margin-left: 3px;
+  width: 100%;
+}
+
+.cv-statement__id {
+  background: #d9d9d9;
+  padding: 4px 6px 4px 6px;
+  font-size: 12px;
+  font-weight: bold;
+  border-radius: 2px;
+  margin-top: 1px;
+}
+
+.cv-statement__icon {
+  color: #50b0a4;
+}
+
+.cv-statement__name {
+  color: #50b0a4;
+  font-weight: bold;
+  font-size: small;
+  margin-top: -2px;
+}
+
+.cv-ifStatement__condition {
+  margin-top: 7px;
+}
+
+.cv-pluginStatement__icon {
+  padding-top: 4px;
+}
+
+.cv-pluginStatement__metricContainer {
+  padding: 4px 8px 4px 8px;
+  border-radius: 2px;
+}
+
+.cv-processorMetric__cpuTime {
+  width: 35px;
+  padding: 2px 4px 2px 8px;
+  border-radius: 2px;
+  margin-top: -2px;
+}
+
+.cv-processorMetric__cpuTimeHighlight {
+  background: #d6b283;
+}
+
+.cv-processorMetric__events {
+  width: 150px;
+  border-radius: 2px;
+  padding: 2px 4px 2px 4px;
+  margin-top: -2px;
+}
+
+.cv-processorMetric__eventMillis {
+  width: 80px;
+  padding: 2px 4px 2px 4px;
+  margin-top: -2px;
+}
+
+.cv-spaceContainer {
+  background-color: white;
+}
+
+.cv-queue__messageText {
+  margin-left: 25px;
+  color: #838383;
+}
+
+.cv-list {
+  display: flex;
+  min-height: 32px;
+}
+
+.cv-listParent li:nth-child(2n+1) {
+  background: #f7f7f7;
+}
+
+.cv-ifElseStatement__title {
+  margin-top: 2px;
+}
+
+.cv-ifElseStatement__name {
+  font-weight: bold;
+}
+
+.cv-ifElseStatement__toggle {
+  margin-top: 3px;
+  color: #898989;
+}
```

Comment:
I'm assuming you ran these CSS changes by the design folks or they themselves suggested them?